hcl2template/types.build.hcp_packer_registry.go

```diff
@@ -0,0 +1,55 @@
+package hcl2template
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
+	packerregistry "github.com/hashicorp/packer/internal/packer_registry"
+)
+
+type HCPPackerRegistryBlock struct {
+	// Bucket description
+	Description string
+	// Bucket labels
+	Labels map[string]string
+
```

Comment:
I think that we should add a bucket slug here.

Reply:
I like the idea. I'm adding it now

Reply:
I'll keep the logic of env variable overriding config values to keep it consistent with the other env variables of packer